ui/app/src/views/ViewDashboard.tsx

```diff
@@ -156,6 +156,9 @@ function ViewDashboard() {
                   dashboardProject={data.metadata.project}
                 />
               }
+              emptyDashboard={
+                <EmptyDashboard additionalText="In order to create a new dashboard. You need to add at least one panel!" />
```

Comment:
Taking this message that was in a snackbar and putting it in the empty state where it's more front and center.